pyof/v0x04/common/action.py

```diff
@@ -298,7 +298,8 @@ class ActionOutput(ActionHeader):
 
     _allowed_types = ActionType.OFPAT_OUTPUT,
 
-    def __init__(self, port=None, max_length=None):
+    def __init__(self, port=None,
```

Comment:
Let me understand a decision here. When `port=None` is packed, it becomes zero. Two questions:
1. Will it be `None` again after unpacking?
2. Does `None` have a special meaning or we can use zero as default as well?

Reply:
Well, `port` here is an `UBInt32`, so packing it with `None` will result in an exception. This is intentional: If you are creating an Output Action, it makes sense that you MUST define the port to output. I think there is no sensible default value in this case.